### PR TITLE
Update global Context documentation comments

### PR DESCRIPTION
**Pull request by erinha**
We discovered after the 2.1.0 release that the documentation system doesn't like the links we used in the deprecated comments for the global `Context` interface properties. Updating the links so that they work with the typedoc system.

### CODE REVIEW

**Review by JoshuaPartlow — DISMISSED** [review]
Those updates all look good to me.

**Review by TrevorJoelHarris — APPROVED** [review]

**Comment by erinha** [comment]
@JoshuaPartlow I had to resolve a merge conflict and I'm concerned that the new `Context.actionInfo` property's deprecated tag will have the same link issue as the `Context.sharepoint` property. Can you tell if that's the case?

**Review by shrshindeMSFT — APPROVED** [review]

**Review by jadahiya-MSFT — DISMISSED** [review]

**Review by JoshuaPartlow — APPROVED** [review]

**Review by jadahiya-MSFT — APPROVED** [review]

File: packages/teams-js/src/public/interfaces.ts [hunk 1]
```diff
@@ -244,26 +244,27 @@ export interface ActionInfo {
 
 /**
  * @deprecated
- * As of 2.0.0, please use {@link app.Context} instead.
+ * As of 2.0.0, please use the {@link app.Context | app.Context} interface and its updated properties instead.
```
Comment (TrevorJoelHarris) [inline packages/teams-js/src/public/interfaces.ts:247]: I'm surprised we have to specify app.Context twice

File: packages/teams-js/src/public/interfaces.ts [hunk 2]
```diff
@@ -244,26 +244,27 @@ export interface ActionInfo {
...
  *
  * @remarks
- * For more details on the updated {@link app.Context} interface, visit
- * {@link https://docs.microsoft.com/microsoftteams/platform/tabs/how-to/using-teams-client-sdk#updates-to-the-context-interface}.
+ * For more details about the updated {@link app.Context} interface, visit the
```
Comment (TrevorJoelHarris) [inline packages/teams-js/src/public/interfaces.ts:250]: does this also need to be app.Context | app.Context?

File: packages/teams-js/src/public/interfaces.ts [hunk 3]
```diff
@@ -371,87 +373,87 @@ export interface Context {
...
+   * As of 2.0.0, please use {@link app.PageInfo.isFullScreen | app.Context.page.isFullScreen} instead
    *
    * Indication whether the tab is in full-screen mode.
    */
   isFullScreen?: boolean;
 
   /**
    * @deprecated
-   * As of 2.0.0, please use {@link app.Context.team.type} instead
+   * As of 2.0.0, please use {@link app.TeamInfo.type | aopp.Context.team.type} instead
```
Comment (TrevorJoelHarris) [inline packages/teams-js/src/public/interfaces.ts:392]: lol aopp

**Review by TrevorJoelHarris — APPROVED** [review]

**Pull request merged**